qa/pom.xml

```diff
@@ -126,19 +126,23 @@
 
     <dependency>
       <groupId>com.oracle.database.jdbc</groupId>
-      <artifactId>ojdbc8</artifactId>
+      <artifactId>ojdbc10</artifactId>
+      <version>19.21.0.0</version>
     </dependency>
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
+      <version>42.6.0</version>
```

Comment:
Why have you introduced `<version>` here and not in the `narayana-bom-test`

Reply:
Thanks Manuel, I will fix that.